plugins/view-transitions/js/view-transitions.js

```diff
@@ -155,6 +155,7 @@ window.plvtInitViewTransitions = ( config ) => {
 					);
 				} else if (
 					document.body.classList.contains( 'home' ) ||
+					document.body.classList.contains( 'blog' ) ||
```

Comment:
Have you check the different `Transition Animation` instead of fade without the changes?

Reply:
No. I didn't make any changes to the settings.